### DISCUSSION TITLE
Restrict certain users for certain APIs

### DISCUSSION THREAD

**Discussion by githubpiyush** [Sunbird-inQuiry/Community#87] [Q&A]
@krgauraw

How can we restrict users to use certain API at OPA level?

CC :- @aks30

**Comment by krgauraw** [comment]
@githubpiyush OPA is not enabled for inquiry micro-service as of now. You may restrict user based on role mapped to api in kong layer itself. 
If you are planning to use OPA only, Please get in touch with @gandham-santhosh . He may help you with approach to enable OPA layer and define rules as per your requirement.

cc: @sajeshkayyath @aks30
